misc/aqp2/big_random.R

```diff
@@ -1,62 +1,88 @@
+
+library(data.table)
 library(aqp)
 library(magrittr)
-library(data.table)
-
-# build the data
-system.time(bigrandom <- lapply(1:1000001, random_profile))
-# user  system elapsed 
-# 933.54    1.95  939.94 
-
-# save for later
-save(bigrandom, file="bigrandom.Rda")
+#
+# # build the data
+# system.time(bigrandom <- lapply(1:1000001, random_profile))
+# # user  system elapsed
+# # 933.54    1.95  939.94
+#
+# # save for later
+#
+load(file = "~/bigrandom.Rda")
```

Comment:
This is in /misc so not a big deal, but `save` / `load`? and indexing to $HOME?

Reply:
Yeah -- sorry -- not sure what you are asking there. This is just a byproduct of me commiting changes between my linux machine and work and not consistently fixing the paths. The latest version should just reference "bigrandom.Rda" unless I messed up.

Reply:
Hah, ok. I had a couple of free minutes. I'll wait a day or two next time.

Reply:
Its OK, i should have committed the changes I had on deck. I didn't think I had hit the review button, but was planning to.